modules/ref-inference-endpoints.adoc

```diff
@@ -212,8 +212,8 @@ endif::[]
 * link:https://docs.openvino.ai/2023.3/ovms_docs_rest_api_kfs.html[OpenVINO KServe-compatible REST API documentation^]
 * link:https://platform.openai.com/docs/api-reference/introduction[OpenAI API documentation^]
 ifdef::upstream[]
-* link:{odhdocshome}/serving-models/#ref-supported-runtimes_serving-large-models[Supported runtimes]
+* link:{odhdocshome}/serving-models/#supported-runtimes_serving-large-models[Supported runtimes]
```

Comment:
[Supported model-serving runtimes] instead of [Supported runtimes] (unless this was intentional).